src/react/deckgl.js

```diff
@@ -158,6 +164,7 @@ export default class DeckGL extends React.Component {
       debug,
       viewport: {x: 0, y: 0, width, height},
       onRendererInitialized: this._onRendererInitialized,
+      onFrameDrawn: this._onFrameDrawn,
```

Comment:
We used to have `onBeforeRender` and `onAfterRender` callbacks in the WebGL renderer but they were recently removed during a cleanup. Looks like we are adding one of them back now, which is fine if we need it.